### ISSUE TITLE
Record currently active monitor (or allow changing the monitor to be recorded while recording)

### ISSUE THREAD

**Issue by hotice** [MaartenBaert/ssr#337]
Would it be possible to have an option to record the currently active monitor (so it would switch between monitors/screens and record the one that has the mouse cursor or active window) or allow changing the recorded screen while recording?

That would be very useful when trying to show multi-monitor features in the screencast.

**Comment by MaartenBaert** [comment]
Good idea, and pretty easy to implement. I've added it to the todo list.

**Comment by claudchereji** [comment]
This would be a very useful feature for tutorials. When recording the screen on an ultra-wide it makes it very hard for the end user to see everything on if they're using a standard sized display. 

I would love to see this feature implemented!
